R-package/R/lgb.cv.R

```diff
@@ -16,10 +16,8 @@ CVBooster <- R6Class(
   )
 )
 
-#' Main CV logic for LightGBM
-#'
-#' Main CV logic for LightGBM
-#'
+#' @title Main CV logic for LightGBM
+#' @name lgb.cv
```

Comment:
this and `lgb.train` were getting grouped into a single documentation object with interwoven fields and R CMD CHECK complained about duplicated parameters. Using explicit `@name` elements avoids this